index.html

```diff
@@ -0,0 +1,141 @@
+      <p>Window Controls Overlay addresses this challenge by introducing a set of technologies throughout JS and CSS that allow to the web content area to expand over the title bar area, remaining only the area corresponding to the overlay that houses the controls of the window.</p>
+
+    </section>
+    <section>
+      <h2>Extensions to the `Navigator` interface</h2>
+      <p>
+        The [[HTML]] specification defines the <dfn>Navigator</dfn> interface, which this
+        specification extends:
+      </p>
+      <pre class="idl">
+        [SecureContext, Exposed=(Window)]
+        partial interface Navigator {
+          [SameObject] readonly attribute WindowControlsOverlay windowControlsOverlay;
+        };
+      </pre>
+      <section>
+        <h3>`windowControlsOverlay` attribute</h3>
+      </section>
+      <p>The {{navigator.windowControlsOverlay}} attribute is an instance of
+        {{WindowControlsOverlay}}. This attribute provides the current visibility of the overlay, the bounding rectangular area of the overlay, and whether there was an ongeometrychange event to that area of the overlay.
+        The <a>user agent</a> MUST run the <a>update the overlay area
+        information</a> algorithm steps to initialize the values and return
+        the {{Navigator.windowControlsOverlay}} attribute.</p>
+    </section>
+    <section data-dfn-for='WindowControlsOverlay' data-link-for=
+    'WindowControlsOverlay'>
+      <h2>
+        <dfn>WindowControlsOverlay</dfn> interface
+      </h2>
+      <pre class='idl' data-cite="DOM HTML">
+        [Exposed=Navigator]
+        interface WindowControlsOverlay : EventTarget {
+          readonly attribute boolean visible;
+          readonly attribute DomRect getBoundingClientRect;
+          attribute EventHandler ongeometrychange;
+        };
+      </pre>
+      <section>
+        <h3>The <dfn>visible</dfn> attribute</h3>
+        <p>The `visible` attribute is a boolean that determines if the window controls overlay has been rendered.</p>
+      </section>
+      <section>
+        <h3>The <dfn>getBoundingClientRect</dfn> method</h3>
+        <p>The `getBoundingClientRect()` method returns a DOMRect that represents the area under the window controls overlay. Web content SHOULD not be displayed beneath the overlay.</p>
+      </section>
+      <section>
+        <h3>The <dfn>ongeometrychange</dfn> attribute</h3>
```

Comment:
Do you know if you need to include the idl for the `ongeometrychange` event? If so, it's defined here: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/frame/window_controls_overlay_geometry_change_event.idl

It probably should be added since it also sends info about the updated rect when it fires:
```
[
    Exposed=Window
] interface WindowControlsOverlayGeometryChangeEvent : Event {
    constructor(DOMString type, WindowControlsOverlayGeometryChangeEventInit eventInitDict);
    [SameObject] readonly attribute DOMRect boundingRect;
    readonly attribute boolean visible;
};
```